browser/ui/brave_browser_command_controller.cc

```diff
@@ -664,6 +666,15 @@ bool BraveBrowserCommandController::ExecuteBraveCommandWithDisposition(
     case IDC_BREAK_TILE:
       brave::BreakTiles(&*browser_);
       break;
+    case IDC_CLOSE_TAB:
+      {
+        int activeTabIndex = browser_->tab_strip_model()->active_index();
```

Comment:
nit: we use snake_case for local variable.

```suggestion
        const int active_tab_index = browser_->tab_strip_model()->active_index();
```

Reply:
okay! will keep it in mind.